transport/spi/src/testFixtures/kotlin/Utils.kt

```diff
@@ -71,12 +73,12 @@ fun startReceiver(configManager: ConfigManager): LinkedBlockingQueue<Message<Orc
         queue.offer(message)
     }
 
-    Thread {
+    thread {
```

Comment:
Cross-linking [this discussion](https://github.com/eclipse-apoapsis/ort-server/pull/886#discussion_r1728707065) for reference.